lms/envs/common.py

```diff
@@ -714,7 +714,16 @@
     # .. toggle_tickets: https://openedx.atlassian.net/browse/SOL-1325
     'ENABLE_OPENBADGES': False,
 
-    # Enable LTI Provider feature.
+    # .. toggle_name: FEATURES['ENABLE_LTI_PROVIDER']
+    # .. toggle_implementation: DjangoSetting
+    # .. toggle_default: False
+    # .. toggle_description: When set to True, Open edX site can be used as an LTI Provider to other systems
+    #    and applications.
+    # .. toggle_warnings: After enabling this feature flag you need to run database migrations which will create
+    #    database tables required for LTI provider functionality.
+    # .. toggle_use_cases: open_edx
+    # .. toggle_creation_date: 2015-06-27
+    # .. toggle_tickets: https://github.com/edx/edx-platform/pull/8620
```

Comment:
How did you come to choose this PR? I think there are older ones than this that might be more helpful in understanding the feature.

Reply:
Changed PR link to the first commit where this flag was introduced.